todo_app/app.py

```diff
@@ -1,7 +1,7 @@
 from todo_app.data.ViewModel import ViewModel
 from flask import Flask, redirect, render_template, request, url_for
 
-from todo_app.data.trello_items import (add_item, get_items,
+from todo_app.data.db_items import (add_item, get_items,
```

Comment:
Nice - the fact this is such a small change is a great sign that you picked a solid interface design earlier 👍